proto/rill/ai/v1/ai.proto

```diff
@@ -42,3 +42,11 @@ message CompletionMessage {
   string data = 2; // Deprecated in favor of content
   repeated ContentBlock content = 3;
 }
+
+message CompletionMessageContext {
+  string explore = 1;
+  string time_range = 2;
```

Comment:
Is this always time syntax expression? Should it be a `TimeRange` proto type like in the metrics queries? Or is there no need?